python-template/{{cookiecutter.placeholder_repo_name}}/tox.ini

```diff
@@ -65,4 +65,5 @@ deps =
     -r{toxinidir}/requirements/quality.txt
 commands =
     make quality
+    twine check {distdir}/*
```

Comment:
It feels like this should go into the `quality` make target instead.  And I think we need to explicitly build the packages before checking them.

Reply:
Done.